c/src/ProtonConfig.cmake.in

```diff
@@ -52,26 +52,38 @@ if (Proton_USE_STATIC_LIBS)
     set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH_OLD})
 endif()
 
-set (Proton_INCLUDE_DIRS @PACKAGE_INCLUDE_INSTALL_DIR@)
+set_and_check (Proton_INCLUDE_DIRS @PACKAGE_INCLUDE_INSTALL_DIR@)
```

Comment:
What does set_and_check do? I can't find any documentation for it in the cmake docs and it isn't anywhere else in the proton code.

Reply:
`set_and_check` is documented in https://cmake.org/cmake/help/latest/module/CMakePackageConfigHelpers.html, they say it should be used for variables that hold paths; I know it is unrelated, but it is also not worth its own commit, seemed to me; I can split it out as its own PR if you think it helps

Reply:
and, yeah, I forgot that gh ci does not have centos, I need to check old cmake can do this

Reply:
I was just about to query whether it is supported on cmake 2.8.12!